bip-musig2.mediawiki

```diff
@@ -440,27 +467,27 @@ In order to produce a partial signature for an X-only public key that is an aggr
 
 <poem>
 The following elliptic curve points arise as intermediate steps in the MuSig2 protocol:
-• ''P<sub>i</sub>'' as computed in ''KeyAggInternal'' is the point corresponding to the ''i''-th signer's X-only public key. Defining ''d'<sub>i</sub>'' to be the ''i''-th signer's secret key as an integer, i.e. the ''d' '' value as computed in the ''Sign'' algorithm of the ''i''-th signer, we have
+• ''P<sub>i</sub>'' as computed in ''KeyAgg'' is the point corresponding to the ''i''-th signer's X-only public key. Defining ''d'<sub>i</sub>'' to be the ''i''-th signer's secret key as an integer, i.e. the ''d' '' value as computed in the ''Sign'' algorithm of the ''i''-th signer, we have
     ''P<sub>i</sub> = with_even_y(d'<sub>i</sub>⋅G) ''.
-• ''Q<sub>0</sub>'' is an aggregate of the signers' public keys and defined in ''KeyAggInternal''  as
+• ''Q<sub>0</sub>'' is the aggregate of the signers' public keys and defined as ''Q'' in ''KeyAgg'':
```

Comment:
```suggestion
• ''Q<sub>0</sub>'' is the aggregate of the signers' public keys and defined like ''Q'' in ''KeyAgg'':
```

Not sure if this is grammatical but it took me a while to parse the current sentence because I expect a definition after "is defined as"... We could also simply rename Q to Q0 in KeyAgg. (I mean, KeyAgg now really outputs the first key, so Q0 makes sense.)

Reply:
It seems slightly confusing to have `Q<sub>0</sub>` in the algorithm description but no `Q` with another subscript. Tried improving the wording.